test/module.mk

```diff
@@ -198,7 +198,8 @@ endif
 ifeq ($(WITH_MRSSM2),yes)
 TEST_SRC += \
 		$(DIR)/test_MRSSM2_gmm2.cpp \
-		$(DIR)/test_MRSSM2_l_to_lgamma.cpp
+		$(DIR)/test_MRSSM2_l_to_lgamma.cpp \
+		$(DIR)/test_MRSSM2_l_to_l_conversion.cpp
```

Comment:
we shouldn't bring in conversion test

Reply:
@wkotlarski , yes, you are right, I am removing it.